### DISCUSSION TITLE
Espulsione di Nicolas

### DISCUSSION THREAD

**Discussion by RizziLucaH** [Frati-Paoloni/Azienda_Vitivinicola#103] [Q&A] [answered]
Secondo voi è giusto brasare ed emarginare Nicolas dai futuri progetti lavorativi?
Io penso che integrarlo nella società sarebbe un errore, risulterebbe una sanguisuga attaccata alle nostre tette dorate, non farebbe un cazzo e percepirebbe una quota come socio.
Aiutatemi a chiarirmi le idee.

**Comment by TomPaglia** [comment, marked as answer]
Bravo Luca! Concordo pienamente con te, mi sono sentito come un ebreo in un campo di concentramento o come un negro in un campo di cotone a lavorare, e in tutto questo nico stava beato nella sua oasi urbana di phpMyAdmin.
#NICOOUT
